### ISSUE TITLE
[Bug]: Investor Section - Only one wallet seems to be working correctly

### ISSUE THREAD

**Issue by jonsnowpt** [Betarena/scores#1888]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### 🐞 Describe the bug

I tried Coinbase wallet and got an immediate transfer error:

![Screenshot 2023-12-27 at 09 35 21](https://github.com/Betarena/scores/assets/37311649/4239a794-279f-44f3-a199-6a832970b914)

It may be related to insufficient funds, but the change of chain was not requested, and also, without balance, the buy option should verify and inform the user.

### 🤔 Expected Behavior

Verify the investor option with other wallets.

### What environment is the bug/issue detected in.

- [ ] Production (scores.betarena.com)
- [X] Development

### What browsers are you seeing the problem on?

- [ ] Firefox
- [X] Chrome
- [ ] Safari
- [ ] Microsoft Edge
- [ ] Brave
- [ ] Opera

### What device/platform can the problem be seen?

- [ ] 📱 Mobile
- [ ] 💻 Tablet
- [ ] 🖥️ Laptop/Desktop

**Comment by migbash** [comment]
@jonsnowpt Did you use the `Coinbase wallet` as an extension ?

**Comment by jonsnowpt** [comment]
@migbash 

Yes.

**Comment by jonsnowpt** [comment]
@migbash 

On mobile there is no opportunity to use another wallet besides Metamas, we need to change the approach to allow other wallets to be also used.

**Comment by migbash** [comment]
**Caputred Logs**

When attempting to use the `Desktop (Brave/Chrome) Coinbase Wallet Extension`:

![Image](https://github.com/Betarena/scores/assets/20924663/5f06d919-053f-443d-a8ce-e27894522b31)

**Comment by migbash** [comment]
> [!NOTE]
> For the issue with the `mobile`, possible main options to solve _this_ issue, are to individually open target `In-App` browsers for the `Web3` wallet selected by the user.

**Comment by migbash** [comment]
> [!Note]
> Corrected, awaiting `🚀 deployment`.

**Comment by jonsnowpt** [comment]
@migbash 

About Coinbase wallet, in this case, can't we forward the user to the in-app browser like Gempad does?

About the rest of the wallets we might need to test afterwards with the real thing, send small amounts with 3 or 4 main wallets featured.

**Comment by migbash** [comment]
> About Coinbase wallet, in this case, can't we forward the user to the in-app browser like Gempad does?

@jonsnowpt No, unfortunately, because we are using exclusively `WalletConnect` now, and it's not appears to be working. 

What you're talking about is the first [1] solution I presented in my above comments, that involved the change of the `authentication` and add other providers/wallets for this to make it work correctly.

As meantioned, this is a issue with WalletConnect and we have moved to the latest version, and it still not working correctly. Only wait until they provide this fix themselves.
